server/build/docker/keycloak/README.md

```diff
@@ -18,44 +18,12 @@ To use this keycloak image, we suggest you to use this configuration settings:
 - First Name Attribute: `firstName`
 - Last Name Attribute: `lastName`
```

Comment:
From my testing, this seems wrong. The settings that worked for me are
```suggestion
- First Name Attribute: `givenName`
- Last Name Attribute: `surname`
```

Can you please double-check what the correct setting is?

Reply:
I can double check

Reply:
Yeah those are correct, added to the README